content/en/docs/setup/platform-setup/kind/index.md

```diff
@@ -0,0 +1,127 @@
+---
+title: KIND
+description: Instructions to setup KIND for Istio.
+weight: 21
+skip_seealso: true
+aliases:
+    - /docs/setup/kubernetes/prepare/platform-setup/kind/
+    - /docs/setup/kubernetes/platform-setup/kind/
+keywords: [platform-setup,kubernetes,kind]
+---
+
+[KIND](https://kind.sigs.k8s.io/) is a tool for running local Kubernetes clusters using Docker container `nodes`.
+KIND was primarily designed for testing Kubernetes itself, but may be used for local development or CI.
+Follow these instructions to prepare KIND for Istio installation.
+
+## Prerequisites
+
+- Please use latest Go version, ideally go 1.13 or greater.
```

Comment:
You capitalize Go once and lower case it another. I believe it should be uppercased.